src/java.ts

```diff
@@ -62,12 +60,11 @@ export async function serverSupportsJREProvisioning(
 
 async function fetchLatestSupportedJRE(properties: ScannerProperties, platformInfo: PlatformInfo) {
   const serverUrl = properties[ScannerProperty.SonarHostUrl];
-  const token = properties[ScannerProperty.SonarToken];
 
   const jreInfoUrl = `${serverUrl}/api/v2/analysis/jres?os=${platformInfo.os}&arch=${platformInfo.arch}`;
   log(LogLevel.DEBUG, `Downloading JRE from: ${jreInfoUrl}`);
```

Comment:
I think we can also take advantage of the `params` config object for requests that use query params. and we can keep the baseUrl, URL and query all separate 

```
params: {
    ID: 12345
  },
```